### PR TITLE
First Version

### PR DESCRIPTION
**Pull request by marians**
This is a Dockerfile for an automated build of the `circlecli` utility.

Once this is merged, https://github.com/giantswarm/giantswarm/wiki/Setting-up-CircleCI#setting-environment-variables-via-a-cli becomes easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # circlecli-docker
-Docker image for the circlecli utility
+
+Docker image for the [`circlecli`](https://github.com/TheRealJoeLinux/circlecli) utility. Automatically built as `giantswarm/circlecli`.
```
Comment (JosephSalisbury) [inline README.md]: How is this building?

Comment (marians) [inline README.md]: The plan is that I set up an automated build on Docker hub

Comment (JosephSalisbury) [inline README.md]: Why not circle?

Comment (marians) [inline README.md]: Because it's much simpler to just use a free automated build and it doesn't use our paid resources

Comment (JosephSalisbury) [inline README.md]: i'd much prefer to have all our builds in one place

Comment (JosephSalisbury) [inline README.md]: Oh, you're putting this on the docker hub. okay, whatever

**Comment by marians** [comment]
> ... whatever

Taking this as a LGTM

**Pull request merged**